docs/development_guide.md

```diff
@@ -0,0 +1,119 @@
+# Development Guide
+
+## How to Contribute
+
+[Contribution Guide](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/CONTRIBUTING.md)
+
+### Code reviews
+
+All submissions, including submissions by project members, require review. We
+use GitHub pull requests for this purpose. Consult
+[GitHub Help](https://help.github.com/articles/about-pull-requests/)
+for more information on using pull requests.
+
+
+## Setup
+
+### Fork the repository on Github
+Visit the
+[gcp-variant-transforms repository](https://github.com/googlegenomics/gcp-variant-transforms)
+to create your own fork of the repository.
+
+### Setup dev environment
+
+#### Clone the forked repository
+```bash
+git clone git@github.com:<username>/gcp-variant-transforms.git
+cd gcp_variant_transforms
+```
+You will also googlegenomics repository as a remote so that you can easily pull
+changes later.
+```bash
+git remote add upstream git@github.com:googlegenomics/gcp-variant-transforms.git
+```
+
+#### Setup virtualenv
+```bash
+sudo apt-get install python-pip python-dev build-essential
+sudo pip install --upgrade pip
+sudo pip install --upgrade virtualenv
+virtualenv venv
+. venv/bin/activate
+```
+
+#### Install dependences
+```bash
+python setup.py install
+```
+
+## Making Changes
+### Create a branch in your forked repository
+```bash
+git checkout -b <branch-name> origin/master
+```
+
+Running this command will create a branch named `<branch-name>` and switch
+you to it.
+
+### Testing
+To run all tests:
+
+```bash
+python setup.py test
+```
+
+To run a specific test:
+```bash
+python setup.py test -s <module>.<test class>.<test method>
```

Comment:
actually, i just noticed that it should be 'gcp_variant_transforms.\<module>.\<test class>.\<test method>'

Reply:
Done.